ir/block.go

```diff
@@ -61,6 +61,9 @@ func (block *Block) LLString() string {
 	for _, inst := range block.Insts {
 		fmt.Fprintf(buf, "\t%s\n", inst.LLString())
 	}
+	if block.Term == nil {
+		panic(fmt.Sprintf("block do not have terminator\ncurrent instructions:\n%s<=== insert terminator at here by calling `func (*Block) NewRet(value.Value)`", buf.String()))
```

Comment:
Slight update of error message. Removed reference to `block.NewRet` since the user may which to add any terminator (e.g. `NewCondBr`).

```suggestion
		panic(fmt.Sprintf("missing terminator in basic block %q.\ncurrent instructions:\n%s", block.Name(), buf.String()))
```